docs/src/inflation.md

```diff
@@ -0,0 +1,45 @@
+# Inflation 
+Inflation is a method which systematically adds stochastic perturbations to the parameter ensemble in the form of gaussian noise. 
+Following the standard Kalman update, the parameter ensemble is perturbed in the following manner:
+```math
+    u_{n+1} = u_n + \zeta_{n} \qquad (1)
+```
+
+where `` \zeta_{n} `` is noise drawn from a gaussian distribution. Two distinct forms of inflation are implemented.
+For both implementations, a scaling factor ``s`` is included to extend usage to cases with mini-batching. For mini-batching,
+the scaling factor should be:
+```math
+    s = \frac{|B|}{|C|}
+```
+where `` |B| `` is the mini-batch size and `` |C| `` is the full dataset size, to account for sampling error. 
+
+## Multiplicative Inflation 
+Multiplicative effectively scales parameter vectors in parameter space, such that the perturbed
+ensemble remains in the linear span of the original ensemble. The implementated update equation follows
+[Huang et al, 2022](https://arxiv.org/abs/2204.04386) eqn. 41:
+
+```math
+\begin{aligned}
+    m_{n+1} = m_{n} ; \qquad u^{j}_{n + 1} = m_{n+1} + \sqrt{\frac{1}{1 - s \Delta{t}}} \left(u^{j}_{n} - m_{n} \right) \qquad (2)
+\end{aligned}
+```
+where ``m`` is the ensemble average which remains unchanged after the perturbation.  
```

Comment:
In this way the variance of the ensemble is increased by ... while the mean remains fixed